zone/questmgr.cpp

```diff
@@ -1101,9 +1106,16 @@ uint16 QuestManager::scribespells(uint8 max_level, uint8 min_level) {
 			}
 			initiator->ScribeSpell(spell_id, book_slot);
 			book_slot = initiator->GetNextAvailableSpellBookSlot(book_slot);
+			spells_learned++;
+			spell_learned = true;
 		}
 	}
-	return spell_count;
+
+	if (spell_learned) {
```

Comment:
Could just check `spells_learned > 0` here and ditch the extra var

Reply:
True enough. Will do that for disciplines as well. Variable was just for code legibility.